tool/tbot/config/configtemplate_mongo.go

```diff
@@ -0,0 +1,92 @@
+/*
+Copyright 2022 Gravitational, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package config
+
+import (
+	"context"
+
+	"github.com/gravitational/teleport/api/types"
+	"github.com/gravitational/teleport/lib/auth"
+	"github.com/gravitational/teleport/lib/client/identityfile"
+	"github.com/gravitational/teleport/tool/tbot/identity"
+	"github.com/gravitational/trace"
+)
+
+// defaultMongoPrefix is the default prefix in generated MongoDB certs.
+const defaultMongoPrefix = "mongo"
+
+// TemplateMongo is a config template that generates TLS certs formatted for
+// use with MongoDB.
+type TemplateMongo struct {
+	Prefix string `yaml:"prefix,omitempty"`
+}
+
+func (t *TemplateMongo) CheckAndSetDefaults() error {
+	if t.Prefix == "" {
+		t.Prefix = defaultMongoPrefix
+	}
+
+	return nil
+}
+
+func (t *TemplateMongo) Name() string {
+	return TemplateMongoName
+}
+
+func (t *TemplateMongo) Describe() []FileDescription {
+	return []FileDescription{
+		{
+			Name: t.Prefix + ".crt",
+		},
+		{
+			Name: t.Prefix + ".cas",
+		},
+	}
+}
+
+func (t *TemplateMongo) Render(ctx context.Context, authClient auth.ClientI, currentIdentity *identity.Identity, destination *DestinationConfig) error {
+	dest, err := destination.GetDestination()
+	if err != nil {
+		return trace.Wrap(err)
+	}
+
+	hostCAs, err := authClient.GetCertAuthorities(ctx, types.HostCA, false)
```

Comment:
Starting from Teleport 10, Database Access will use Database CA instead of Host CA (if I understand the logic correctly) where Teleport 9.0 and before are still using Host CA.
```suggestion
	hostCAs, err := authClient.GetCertAuthorities(ctx, types.DatabaseCA, false)
```
PR: https://github.com/gravitational/teleport/pull/9593